.github/workflows/twister.yaml

```diff
@@ -213,6 +213,17 @@ jobs:
           ccache -p
           ccache -z -s -vv
 
+      - name: Update BabbleSim to manifest revision
+        run: |
+          export BSIM_VERSION=$( west list bsim -f {revision} )
+          echo "Manifest points to bsim sha $BSIM_VERSION"
+          cd /opt/bsim_west/bsim
+          git fetch -n origin ${BSIM_VERSION}
+          git config --global advice.detachedHead false
```

Comment:
how about moving this with the other `git config` commands in the `Environment Setup` step? Just so everything git is configured at the start and this step does not have to mix config and normal operations

Reply:
I just changed the command below to use that option for that command alone instead of setting it globally.
(to keep this localized to a single step, as it is replicated in several places and we may want to eventually refactor it out)

Reply:
ok, though I think it would have been fine to change it for all commands, less spammy stuff in the log